app/vep/utils/web_metadata.py

```diff
@@ -24,3 +25,27 @@ def get_vep_support_location(genome_id):
         raise HTTPError
     except Exception as e:
         logging.error(e)
+
+async def get_genome_metadata(genome_id: str) -> GenomeAnnotationProvider:
+    try:
+        response = requests.get(
+            WEB_METADATA_API
+            + "genome/"
+            + genome_id
+            + "/dataset/genebuild/attributes?"
+            + "attribute_names=genebuild.provider_name&"
+            + "attribute_names=genebuild.display_version&"
+            + "attribute_names=genebuild.last_geneset_update"
+        )
+        response.raise_for_status()
+        attributes = {}
+        for attribute in response.json()['attributes']:
+            name = attribute['name']
+            value = attribute['value']
+            attributes[name] = value
+        return attributes
+    except HTTPError as http_error:
+        logging.error(http_error)
+        raise HTTPError
+    except Exception as e:
+        logging.error(e)
```

Comment:
Follow-up to code reaview: please add `raise Excpetion` here (I can't add it myself as I don't have this function in [my PR](https://github.com/Ensembl/ensembl-web-tools-api/pull/42)). Re-raising allows the caller (`get_form_config()`) to handle the original Exception instead of handling the unexpected `None` return value [here](https://github.com/Ensembl/ensembl-web-tools-api/blob/636ac1c9e28f2476074bab536f034b5847db9efb/app/vep/vep_resources.py#L220).